### PR TITLE
feat: adds Lesson6 with implemented code

### PR DESCRIPTION
**Pull request by cibezim**
Implemented code for needed functions

### CODE REVIEW

File: lesson_06/conditionals/conditionals_app/src/main/java/com/codedifferently/lesson6/Lesson6.java [hunk 1]
```diff
@@ -61,7 +89,14 @@ public static String convertGpaToLetterGrade(double gpa) {
    * @return The factorial of n.
    */
   public static int computeFactorial(int n) {
-    return 0;
+    if (n < 0) {
+      return -1;
```
Comment (anthonydmays) [inline lesson_06/conditionals/conditionals_app/src/main/java/com/codedifferently/lesson6/Lesson6.java:93]: This is the wrong behavior for n < 0.

**Review by anthonydmays — CHANGES_REQUESTED** [review]

File: lesson_06/conditionals/conditionals_app/src/main/java/com/codedifferently/lesson6/Lesson6.java (hunk 1 above)
Comment (cibezim) [reply lesson_06/conditionals/conditionals_app/src/main/java/com/codedifferently/lesson6/Lesson6.java]: deleted logic

Comment (anthonydmays) [reply lesson_06/conditionals/conditionals_app/src/main/java/com/codedifferently/lesson6/Lesson6.java]: Actually, the proper behavior is to throw an exception. I'll allow this for now.

**Review by anthonydmays — APPROVED** [review]
LGTM

**Pull request closed without merging**